src/forms/inputs/radio-group.js

```diff
@@ -133,16 +127,12 @@ function RadioGroup(props) {
   } = omitLabelProps(props)
   const optionObjects = serializeOptions(options)
   return (
-    <LabeledField
-      className={className}
-      labelComponent={RadioGroupLegend}
-      {...props}
-    >
-      {optionObjects.map((option, i) => {
+    <LabeledFieldset className={className} {...props}>
```

Comment:
@danparnella what happens if `label` is `false`? Pretty certain `<fieldset>` must have a unique `<legend>`; if a radio group has `label` set to `false`, I believe we would still be rendering a `<fieldset>` but without a `<legend>`?

Reply:
Yep, that's correct. If the design requires `label={false}`, then the developer should use `aria-labelledby` to specify which element to use as the "legend". It might be good for me to add that as an example in the story though.

Reply:
It sounds like Ji ran into some issues where `aria-labelledby` on a fieldset still fails accessibility checks because a legend is **required** when using a `fieldset` ([reference](https://www.w3.org/TR/WCAG20-TECHS/H71.html)). An alternative might be to have `<div role="group" aria-labelledby="">` in this instance.

The reference above is a _technique_ which isn't _required_ to meet WCAG, but is rather a guideline. We might need to dig more into the actual Success Criterion to see if this is a strict requirement or rather a strict interpretation on the accessibility tool. Maybe we could do testing with SiteImprove and Axe's accessibility testing extensions?

@jhp0621 Am I remembering correctly? Do you have more information/resources that you can share?

Reply:
I just did some testing [here](https://q6ownu.csb.app/), and the output is the exact same on VoiceOver. @jhp0621 maybe something to dig into a little deeper with the EA report. Does it list which success criterion are failing?